File: abs/abs.js
// ATOMIC FUNCTION:abs
// TAGS: math，numeric
// LANG: node.js
// FUNCTION: Math.abs(val, precision)

var start_val=-1.2;
var expected_end_val=1.2;

var end_val=Math.abs(start_val);

// TEST
if(end_val==expected_end_val){
    console.log('PASS');
}else{
    console.log('FAIL');
}

File: abs/abs.java
public class abs {
    // ATOMIC_FUNCTION:  abs(val)
    // TAGS: math，numeric
    // LANG:  java
    // FUNCTION: abs(val)



    public static void main(String args[]) {
        double start_val = -1.2;
        double expected_end_val = 1.2;

        double end_val = Math.abs(start_val);

        // TEST
        if(end_val == expected_end_val) {
            System.out.println("PASS");
        } else {
            System.out.println("FAIL");
        }
    }
}

